sklearn/utils/testing.py

```diff
@@ -536,37 +536,9 @@ def uninstall_mldata_mock():
     datasets.mldata.urlopen = urlopen
 
 
-# Meta estimators need another estimator to be instantiated.
-META_ESTIMATORS = ["OneVsOneClassifier", "MultiOutputEstimator",
```

Comment:
Do we think there is much concern that existing implementors have been monkey-patching this??

Reply:
Pretty sure they have. But I'm not too concerned. That was clearly not meant as a public mechanism.